scipy/stats/_multivariate.py

```diff
@@ -2795,3 +2796,109 @@ def __init__(self, dim=None, seed=None):
 
     def rvs(self, size=1, random_state=None):
         return self._dist.rvs(self.dim, size, random_state)
+
+class ortho_group_gen(multi_rv_generic):
+    r"""
+    A matrix-valued O(N) random variable.
+
+    Return a random orthogonal matrix, drawn from the O(N) Haar
+    distribution (the only uniform distribution on O(N)).
+
+    The `dim` keyword specifies the dimension N.
+
+    Methods
+    -------
+    ``rvs(dim=None, size=1, random_state=None)``
+        Draw random samples from O(N).
+
+    Parameters
+    ----------
+    dim : scalar
+        Dimension of matrices
+
+    Notes
+    ----------
+    This class is closely related to special_ortho_group.
```

Comment:
scipy.stats.special_ortgho_group wrapped into single backicks would produce a hyperlink in the html docs